imap_processing/ena_maps/ena_maps.py

```diff
@@ -527,8 +495,89 @@ def project_pset_values_to_map(
                 raise NotImplementedError(
                     "Only PUSH and PULL index matching methods are supported."
                 )
+
             self.data_dict[value_key] += pointing_projected_values
 
+
+class RectangularSkyMap(AbstractSkyMap):
+    """
+    Map which tiles the sky with a 2D rectangular grid of azimuth/elevation pixels.
+
+    ^  |15,  75|45,  75|75,  75|105,  75|...|255,  75|285,  75|315,  75|345,  75|
+    |  |15,  45|45,  45|75,  45|105,  45|...|255,  45|285,  45|315,  45|345,  45|
+    |  |15,  15|45,  15|75,  15|105,  15|...|255,  15|285,  15|315,  15|345,  15|
+    |  |15, -15|45, -15|75, -15|105, -15|...|255, -15|285, -15|315, -15|345, -15|
+    |  |15, -45|45, -45|75, -45|105, -45|...|255, -45|285, -45|315, -45|345, -45|
+    |  |15, -75|45, -75|75, -75|105, -75|...|255, -75|285, -75|315, -75|345, -75|
+    |
+    ---------------------------------------------------------------> Azimuth (degrees)
+    Elevation (degrees)
+
+    Fig. 1: Example of a rectangular grid of pixels in azimuth and elevation coordinates
+    in degrees, with a spacing of 30 degrees. There will be 12 azimuth bins and 6
+    elevation bins in this example, resulting in 72 pixels in the map.
+
+    NOTE: Internally, the map is stored as a 1D array of pixels, and all data arrays
```

Comment:
There is a `Notes` section in numpydoc which might be good here? Or in rest I think you can add a `.. note::` to give a note highlight nearby this too.
https://numpydoc.readthedocs.io/en/latest/format.html#notes

Reply:
Good point - I moved this into a Notes section in the numpydoc style.